File: 10041_Vito's_Family.cpp
#include <bits/stdc++.h>
using namespace std;
void cac(vector<int>& nums){
    sort(nums.begin(),nums.end());
    int n = nums.size();
    int half = n/2;
    int basic = nums[half];
    int ans = 0;
    for(int i = 0 ; i < n ; i++){
        ans+=abs(nums[i] - basic);
    }
    cout << ans << endl;
}
int main(){
    int n;
    cin >> n;
    for(int i = 0 ; i < n ; i++){
        int a;
        cin >> a;
        vector<int> nums(a);
        for(int j = 0 ; j < a;j++){
            cin >> nums[j];
        }
        cac(nums);
    }
}

File: CPE49/10041_Vito's_Family.py
t = int(input())
for _ in range (t):
	inp = list(map(int,input().split()))
	n = inp[0]
	arr = inp[1:n+1]
	arr.sort()
	half = n // 2
	pivot = arr[half]
	ans = 0
	for i in range (n):
		ans += abs(arr[i] - pivot)
	print(ans)